pkg/deployment/cluster_scaling_integration.go

```diff
@@ -253,3 +252,70 @@ func (ci *clusterScalingIntegration) updateClusterServerCount(ctx context.Contex
 	ci.lastNumberOfServers.DBServers = &dbserverCount
 	return safeToAskCluster, nil
 }
+
+// GetLastNumberOfServers returns the last number of servers
+func (ci *clusterScalingIntegration) GetLastNumberOfServers() arangod.NumberOfServers {
+	ci.lastNumberOfServers.mutex.Lock()
+	defer ci.lastNumberOfServers.mutex.Unlock()
+
+	return ci.lastNumberOfServers.NumberOfServers
+}
+
+// DisableScalingCluster disables scaling DBservers and coordinators
+func (ci *clusterScalingIntegration) DisableScalingCluster() error {
+	ci.scaleEnabled.mutex.Lock()
+	defer ci.scaleEnabled.mutex.Unlock()
+
+	// Turn off scaling DBservers and coordinators in arangoDB for the UI
+	ctx := context.Background()
+	if err := ci.depl.SetNumberOfServers(ctx, nil, nil); err != nil {
+		return maskAny(err)
+	}
+
+	ci.scaleEnabled.enabled = false
+	return nil
+}
+
+// EnableScalingCluster enables scaling DBservers and coordinators
+func (ci *clusterScalingIntegration) EnableScalingCluster() error {
+	ci.scaleEnabled.mutex.Lock()
+	defer ci.scaleEnabled.mutex.Unlock()
+
+	if ci.scaleEnabled.enabled {
+		return nil
+	}
+
+	ctx := context.Background()
+	lastNumberOfServers := ci.GetLastNumberOfServers()
```

Comment:
I would prefer the operator to use the acutal values of running dbservers/coodrinators according to spec. This value here might be outdated

Reply:
changed